docs/src/modules/components/AppProductsDrawer.tsx

```diff
@@ -57,17 +57,31 @@ const ProductSubMenu = React.forwardRef<HTMLDivElement, ProductSubMenuProps>(
 
 const LinksWrapper = styled('div')(
   sx({
-    pl: 6.5,
+    pl: 5.5,
     pt: 1,
-    gap: 0.5,
+    // gap: 0.5,
     display: 'flex',
     flexDirection: 'column',
     '& > a': {
-      fontSize: '0.875rem',
-      lineHeight: 1.7,
       display: 'flex',
       justifyContent: 'space-between',
-      '&:hover': { textDecoration: 'underline' },
+      py: 0.5,
+      px: 1,
+      borderRadius: 0.5,
+      fontWeight: 500,
+      fontSize: (theme) => theme.typography.pxToRem(14),
+      color: (theme) =>
+        theme.palette.mode === 'dark' ? theme.palette.primary[300] : theme.palette.primary[600],
+      '&:hover': {
+        backgroundColor: (theme) =>
+          theme.palette.mode === 'dark'
+            ? alpha(theme.palette.primaryDark[700], 0.4)
+            : theme.palette.grey[50],
```

Comment:
@siriwatknp Just to let you know, I was seeing what seems to be a Typescript error because of this part and I don't know how to fix it properly so the checks might failing also because of this 😅 

<img width="300" alt="Screen Shot 2021-12-22 at 13 14 57" src="https://user-images.githubusercontent.com/67129314/147138720-ed234e7f-3fe9-4d80-a7fa-c8271e7b83ba.png">